salt/states/selinux.py

```diff
@@ -144,3 +161,59 @@ def boolean(name, value, persist=False):
         return ret
     ret['comment'] = 'Failed to set the boolean {0} to {1}'.format(name, rvalue)
     return ret
+
+
+def module(name, module_state='Enabled', version='any'):
+    '''
+    Enable/Disable and optionally force a specific version for an SELinux module
+
+    name
+        The name of the module to control
+
+    module_state
```

Comment:
Given that it looks like capitalization matters here, we need to either document that or normalize the argument by lowering it before we process it, I think.